app/assets/stylesheets/two-factor-authentication.scss

```diff
@@ -0,0 +1,5 @@
+// Styles around two-factor authentication
+
+.two-factor-qr {
+  background-color: white;
```

Comment:
- Color `white` should be written in hexadecimal form as `#ffffff`
- Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.